src/VolleyManagement.Backend/VolleyManagement.Crosscutting.Contracts/Providers/ISecretsProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace VolleyManagement.Crosscutting.Contracts.Providers
+{
+    using Microsoft.Extensions.Configuration;
+
+    /// <summary>
+    /// Provides configurations for external auth secrets
+    /// </summary>
+    public interface ISecretsProvider
```

Comment:
How it's different from IConfigurationProvider?

Reply:
Return type of the method is different (can be made the same).
First I thought to put this method into IConfigurationProvider interface but I didn't. My intention was to create a very specific interface(according to interface segregation principle).
Do you think this is overkill?